### PR TITLE
fix GH#10055: setting key signature mode in the inspector crashes

### LINKED ISSUE #10055: [MU4 Issue] Setting key signature mode in the inspector crashes
**Describe the bug**
Setting key signature mode in the inspector crashes.

**Desktop (please complete the following information):**
 - OS: Windows 11

### PR DESCRIPTION
**Pull request by Howard-C**
Fixes #10055.

### CODE REVIEW

File: src/engraving/libmscore/key.h [hunk 1]
```diff
@@ -61,24 +61,6 @@ enum class Key {
...
-    DORIAN,
-    PHRYGIAN,
-    LYDIAN,
-    MIXOLYDIAN,
-    AEOLIAN,
-    IONIAN,
-    LOCRIAN
-};
-
 static inline bool operator<(Key a, Key b) { return static_cast<int>(a) < static_cast<int>(b); }
```
Comment (igorkorsukov) [inline src/engraving/libmscore/key.h:64]: I think these operators need to be moved too

File: src/engraving/property/propertyvalue.cpp [hunk 2]
```diff
@@ -237,7 +237,8 @@ PropertyValue PropertyValue::fromQVariant(const QVariant& v, P_TYPE type)
     case P_TYPE::DYNAMIC_TYPE:     return PropertyValue(DynamicType(v.toInt()));
     case P_TYPE::DYNAMIC_RANGE:    return PropertyValue(DynamicRange(v.toInt()));
     case P_TYPE::DYNAMIC_SPEED:    return PropertyValue(DynamicSpeed(v.toInt()));
-    case P_TYPE::HOOK_TYPE:    return PropertyValue(HookType(v.toInt()));
+    case P_TYPE::HOOK_TYPE:        return PropertyValue(HookType(v.toInt()));
+    case P_TYPE::KEYMODE:          return PropertyValue(KeyMode(v.toInt()));
```
Comment (igorkorsukov) [inline src/engraving/property/propertyvalue.cpp:241]: You also need to add to the `toQVariant` method

Comment (Jojo-Schmitz) [reply src/engraving/property/propertyvalue.cpp:241]: Like up after line 172 add `case P_TYPE::KEYMODE:        return static_cast<int>(value<KeyMode>());`, right?

Comment (igorkorsukov) [reply src/engraving/property/propertyvalue.cpp:241]: right!

File: src/engraving/libmscore/key.h (hunk 1 above)
Comment (Howard-C) [reply src/engraving/libmscore/key.h:64]: But these operators are for `Key`, not `KeyMode`? The definition of `Key` itself has not been moved, should it be moved as well?

File: src/engraving/property/propertyvalue.cpp (hunk 2 above)
Comment (Howard-C) [reply src/engraving/property/propertyvalue.cpp:241]: Done

File: src/engraving/libmscore/key.h (hunk 1 above)
Comment (igorkorsukov) [reply src/engraving/libmscore/key.h:64]: Oh, you're right, I was wrong, no need to move them

**Review by igorkorsukov — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #10055 on Win10, Mac12, LinuxMint 20.2 - FIXED      

Please merge into master

Thanks!

**Pull request merged**